### DISCUSSION TITLE
"decreasing" as attribute for a single case

### DISCUSSION THREAD

**Discussion by researchtool** [jazznbass/scan#36] [Ideas]
Currently `decreasing = TRUE/FALSE` is passed as argument to the analysis functions.

I was wondering if it wouldn't be more consistent to store the information about whether a DV is expected / desired to increase or decrease as an attribute of the single case (or as additional information to the DV).

This could simplify the analysis with DVs of different "directions".

**Comment by jazznbass** [comment]
Ok, sounds like a good idea.

But it is in conflict (or gets to complicated) when you have various dependent variables.
In one of my own studies we had increase in positive behavior and decrease in negative.
Someone who changes the dependent variable with dvar in an analysis must also remember to change the direction.
But then -- you will have to do this in either way.

All relevant functions should then report the direction to prevent errors.

Technically it is not a problem just a bit of work.

**Reply by researchtool** [reply]
mhm. good point. I don't have a got solution, but I'm goning to think about it.
